test/error.test.ts

```diff
@@ -0,0 +1,176 @@
+import { expect } from 'chai';
+import axios from 'axios';
+import server from '../src/index';
+import { prisma } from '../src/app/client/client';
+import 'dotenv/config';
+import bcrypt from 'bcrypt';
+
+const port = process.env.PORT;
+const url = `http://localhost:${port}/`;
+before(async () => {
+  try {
+    await server.listen(port).then(async ({ url }) => {
+      console.log(url);
+    });
```

Comment:
aqui ao invés de utilizar `await` e `then` poderia colocar o console na linha abaixo ao `await`
```suggestion
    const { url } = await server.listen(port)
    console.log(url);
```